modules/redis/redis-grammar.ym

```diff
@@ -44,6 +44,7 @@
 %token KW_REDIS
 %token KW_COMMAND
 %token KW_AUTH
+%token KW_TIMEOUT
```

Comment:
Please add the new keyword to redis-parser.c as well, otherwise it won't work.

Reply:
I completely missed this comment :( Thank you very much! Fixed